src/zope/app/authentication/browser/granting.py

```diff
@@ -106,15 +106,15 @@ def renderSelectedItem(self, index, text, value, name, cssClass):
                              cssClass=cssClass,
                              checked="checked",
                              type='radio',
-                             extra = 'onclick="changeMatrix(this);"')
+                             extra='onclick="changeMatrix(this);"')
         return self._tdTemplate % (tdClass, id, text, elem)
 
     def renderItems(self, value):
         # check if we want to select first item, the previously selected item
         # or the "no value" item.
         if (value == self.context.missing_value
             and getattr(self, 'firstItem', False)
-            and len(self.vocabulary) > 0):
+                and len(self.vocabulary) > 0):
```

Comment:
Can you please indent the line above as well?  It looks weird when the first continuation line is not indented, but the second is.

Reply:
Fixed.